### DISCUSSION TITLE
Fake-08 and WII homebrew?

### DISCUSSION THREAD

**Discussion by Djjaner** [jtothebell/fake-08#134] [Q&A]
Hello, exists chance that this great thing will be available too for Nintendo Wii?

Thanks for reply. Have a nice day.

**Comment by jtothebell** [comment]
Yes, I'd like to add Wii to the supported platforms. I need to get homebrew set up on the vWii of my Wii U because I don't have an actual Wii. I'm not sure when it will happen, but hopefully in the not too distant future.
